osu.Game/Screens/Edit/EditorBeatmap.cs

```diff
@@ -238,6 +248,8 @@ protected override void Update()
 
             if (batchPendingUpdates.Count > 0)
                 UpdateState();
+
+            hasTiming.Value = !ReferenceEquals(ControlPointInfo.TimingPointAt(editorClock.CurrentTime), TimingControlPoint.DEFAULT);
```

Comment:
Is there a particular reason I am missing (I didn't pay attention super closely during the stream) for why this is using `TimingPointAt()` and the clock current time at all, rather than checking count of `ControlPointInfo.TimingPoints`? If it's to be done every frame anyway...

Reply:
I think either is fine. This felt more correct.

Reply:
Not super sure about this either, but let's roll with it for now. My reasoning's a bit different - the read-only-ness of `TimingControlPoint.DEFAULT` is quite flaky.